### PR TITLE
fix test for dvc-render updates to plots title anchor

### PR DESCRIPTION
**Pull request by dberenbaum**
Fixes tests for https://github.com/iterative/dvc-render/pull/160

I don't see any issues in VS Code initially. @mattseddon Any idea whether this would cause problems there or in Studio?

### CODE REVIEW

**Review by skshetry — APPROVED** [review]

**Comment by skshetry** [comment]
Since it's a small fix on dvc side, I am merging this to unblock CI. 

I'll wait for @mattseddon to respond before making a release. We can fix it accordingly.

**Comment by mattseddon** [comment]
should be fine

**Pull request merged**
